docs/manual/java/guide/broker/KafkaClient.md

```diff
@@ -48,3 +51,26 @@ akka.kafka.producer.kafka-clients {
 }
 ```
 
+## Consuming only Services
+
+When implementing a Lagom Service that will only consume from a Kafka Topic you can import the Lagom Kafka Client alone (instead of importing the Lagom Kafka Broker and a Lagom Persistence implementation).
```

Comment:
While "consumer" is a standard EIP/ESB term, maybe using the word "subscriber" would fit better here?